spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe 'User Show Page', type: :feature do
+  before do
+    @user = User.create(name: 'Abdu', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Yemen.',
+                        posts_counter: 1)
+
+    visit user_path(@user)
+  end
+
+  it 'shows the user details' do
+    expect(page).to have_selector("img[src*='#{@user.photo}']")
+    expect(page).to have_content(@user.name)
+    expect(page).to have_content("Number of Posts:#{@user.posts_counter}")
+    expect(page).to have_content(@user.bio)
+  end
```

Comment:
- [ ] Please kindly add the missing test highlighted below 👇 

<img width="431" alt="deel" src="https://github.com/GeekyHacks/FlightOperationsBlog/assets/103386183/a1d61c05-3b87-4cf1-9a4c-431fba8baabb">